react/react1/week3/homework/todo-app/src/components/AddTodo.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+
+const AddTod = ({ addTodo, items }) => {
+  const newId = Math.floor(Math.random() * 10000);

```

Comment:
In the projects it is not a good idea to use random number for ids. You can get last item id from todolist, you can add 1 to it and you can set it when you call this component.
```suggestion
const AddTod = ({ addTodo, items, newId }) => {
```

Reply:
yes thanks. changes implemented.